### PR TITLE
Quicklook map docs

### PR DESCRIPTION
**Pull request by nekhtiari**
Add Quicklooks mapping jupyter example and update catalog docs

### CODE REVIEW

File: examples/guides/catalog.ipynb [hunk 1]
```diff
@@ -89,7 +89,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "catalog.plot_quicklooks(figsize=(20,20))"
+    "catalog.plot_quicklooks(scenes=search_results, aoi=aoi)"
```
Comment (rodrigoalmeida94) [inline examples/guides/catalog.ipynb:92]: "map_quicklooks" I suppose

File: examples/guides/catalog.ipynb [hunk 2]
```diff
@@ -102,9 +102,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "up42-py",
+   "display_name": "up42-pyapi-dev",
    "language": "python",
-   "name": "up42-py"
+   "name": "up42-pyapi-dev"
```
Comment (rodrigoalmeida94) [inline examples/guides/catalog.ipynb:107]: I guess this you should revert, always a mess with the jupyter notebooks

File: mkdocs.yml [hunk 3]
```diff
@@ -38,6 +38,7 @@ nav:
         - Radar processing: examples/radar_processing_1.ipynb
         - Airport monitoring: examples/airports-parallel.ipynb
         - Flood mapping: examples/flood_mapping.ipynb
+        -Quicklooks mapping: examples/map-quicklooks.ipynb
```
Comment (rodrigoalmeida94) [inline mkdocs.yml:41]: Extra space after -

**Review by rodrigoalmeida94 — APPROVED** [review]

File: mkdocs.yml (hunk 3 above)
Comment (nekhtiari) [reply mkdocs.yml:41]: oh yes

File: examples/guides/catalog.ipynb (hunk 2 above)
Comment (nekhtiari) [reply examples/guides/catalog.ipynb:107]: oh, ok

File: examples/guides/catalog.ipynb (hunk 1 above)
Comment (nekhtiari) [reply examples/guides/catalog.ipynb:92]: :D right

File: mkdocs.yml (hunk 3 above)
Comment (chrieke) [inline mkdocs.yml]:
```suggestion
        - Quicklooks mapping: examples/map-quicklooks.ipynb
```

**Review by chrieke — COMMENTED** [review]
LGTM, thanks for the readme fixes. Did it also work in the guides catalog notebook?

File: mkdocs.yml (hunk 3 above)
Comment (nekhtiari) [reply mkdocs.yml]: I already fix this, thanks

**Comment by nekhtiari** [comment]
> LGTM, thanks for the readme fixes. Did it also work in the guides catalog notebook?

Yes, I tired and it worked!

**Pull request merged**